src/r_voxel.c

```diff
@@ -803,7 +803,7 @@ static void VX_DrawColumn (vissprite_t * spr, int x, int y)
 	byte * dest = I_VideoBuffer + viewwindowy * linesize + viewwindowx;
 
 	// iterate over screen columns
-	fixed_t ux = (((int64_t)Ax - 1) | (FRACUNIT - 1)) + 1;
+	uint64_t ux = ((Ax - 1) | (FRACUNIT - 1)) + 1;
```

Comment:
Unfortunately I still get flickering here (turn view slowly): [woofsav0.zip](https://github.com/fabiangreffrath/woof/files/13985964/woofsav0.zip)

Also, this causes glitches on the left side of the screen when a voxel reaches the edge.